README.md

```diff
@@ -17,3 +17,23 @@ corresponding video.
 #### Saved notes
 
 Notes are saved per video.
+
+#### `.env` file
+
+Need to have these keys before launching the app. First two are app specific token.
+Sharing privately. Last one needs to be generated by the user himself.
+
+REACT_APP_DROPBOX_KEY=some_key
+REACT_APP_DROPBOX_SECRET=some_secret
+REACT_APP_DROPBOX_SOUMIKS_ACCESS_TOKEN=some_access_token
```

Comment:
Why is this named `SOUMIKS_ACCESS_TOKEN` ?

Reply:
Forgot to update the keys in readme. Was using this name before but have changed it.